src/Serilog.Sinks.AmazonKinesis/Sinks/AmazonKinesis/HttpLogShipper-net40.cs

```diff
@@ -285,10 +285,8 @@ static bool TryReadLine(Stream current, ref long nextStart, out string nextLine)
 
             current.Position = nextStart;
 
```

Comment:
Maybe add a comment: 

"Important not to dispose this StreamReader as the stream must remain open."